lib/bitcoin/BitcoinProcessor.ts

```diff
@@ -214,40 +215,38 @@ export default class BitcoinProcessor {
   /**
    * Writes a Sidetree transaction to the underlying Bitcoin's blockchain.
    * @param anchorString The string to be written as part of the transaction.
-   * @param fee The fee to be paid for this transaction.
+   * @param minFee The minimum fee to be paid for this transaction.
    */
-  public async writeTransaction (anchorString: string, fee: number) {
-    console.info(`Fee: ${fee}. Anchoring string ${anchorString}`);
-    // ----
-    // Issue #347 opened to track the investigation for this hardcoded value.
-    // Ensure that we are always paying this minimum fee.
-    fee = Math.max(fee, 1000);
-    // ----
+  public async writeTransaction (anchorString: string, minFee: number) {
```

Comment:
This changes the contract between core and bitcoin API and needs documentation updates. https://github.com/decentralized-identity/sidetree/blob/master/docs/implementation.md#write-a-sidetree-transaction